rules/integrations/o365/impact_microsoft_365_potential_ransomware_activity.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2021/07/15"
+maturity = "development"
+updated_date = "2021/07/15"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when Microsoft Cloud App Security reported when a user uploads files to the cloud that might be infected with ransomware.
+"""
+false_positives = ["Unknown"]
```

Comment:
False positive Ideas:
`If Cloud App Security identifies, for example, a high rate of file uploads or file deletion activities it may represent an adverse encryption process.`

https://docs.microsoft.com/en-us/cloud-app-security/anomaly-detection-policy